src/Model/Contact.php

```diff
@@ -1245,10 +1245,10 @@ public static function isHidden($cid)
 	/**
 	 * Returns posts from a given contact url
 	 *
-	 * @param string $contact_url Contact URL
-	 * @param bool   $thread_mode
-	 * @param int    $update
-	 * @param int    $parent
+	 * @param string  $contact_url Contact URL
+	 * @param bool    $thread_mode
+	 * @param integer $update      Update mode 
```

Comment:
Please use the more ubiquitous `int`, or at least the same type string in the same doc block.